contracts/NFTStorefront.cdc

```diff
@@ -122,36 +130,58 @@ pub contract NFTStorefront {
         pub var purchased: Bool
         // The Type of the NonFungibleToken.NFT that is being listed.
         pub let nftType: Type
-        // The ID of the NFT within that type.
+        // The Resource ID of the NFT which can only be set in the contract
+        //TODO how can this better be enforced?
+        pub let nftUUID: UInt64
+        // The Type of the FungibleToken that payments must be made in.
         pub let nftID: UInt64
         // The Type of the FungibleToken that payments must be made in.
         pub let salePaymentVaultType: Type
         // The amount that must be paid in the specified FungibleToken.
         pub let salePrice: UFix64
         // This specifies the division of payment between recipients.
         pub let saleCuts: [SaleCut]
-
+        // This lets teams using this contract easily distinguish events that are relevent to them
```

Comment:
how does a customID do this? Is this like a Tag?

Reply:
@DavidOrchard yeah, you can think of it as a Tag. Having the `customID` included on the the events being emitted means that teams don't necessarily need to query their backend to know if the listing is relevant to them.

Reply:
I could see us using this in a couple ways:
-Identifying the marketplace (eg. on NFL we have an external and internal MP that share the Storefront contract but we can differentiate between where the listing came from)
-Adding the buyer address